### PR TITLE
Added riddles and answers

### LINKED ISSUE #19: Write riddles
We need riddles for part of the challenges in the game.

They should be medium difficulty, ideally (but not necessarily and definitely not all) humorous.  We should have more than we strictly need; this way we can randomise what gets asked improving replayability.  Also, if a question is answered wrong, we can offer a different riddle, rather than letting them reguess

Create an CSV asset of the following structure:
```csv
question_text, correct_answer, incorrect_answer1, incorrect_answer2, incorrect_answer3
```

We'll deal with reading them into the application in another issue

### PR DESCRIPTION
**Pull request by PhantomRaviolis**
Resolves #19

### CODE REVIEW

File: assets/csv/riddles.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"What has a face and two hands but no arms or legs?", "Clock"
+"What five-letter word becomes shorter when you add two letters to it?", "Shorter"
```
Comment (bcvery1) [inline assets/csv/riddles.csv]: Answer should be "Short"

File: assets/csv/riddles.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+"What word begins and ends with an E but only has one letter?", "Envelope"
+"What has a neck but no head?", "Bottle"
+"What gets wetter as it dries?", "Sponge"
```
Comment (bcvery1) [inline assets/csv/riddles.csv]: Isn't this "Towel"?

Comment (PhantomRaviolis) [reply assets/csv/riddles.csv]: Can be either I guess

Comment (PhantomRaviolis) [reply assets/csv/riddles.csv]: Could remove it to save confusion?

Comment (bcvery1) [reply assets/csv/riddles.csv]: Change it to towel - more like answer, sponges don't really dry things

**Pull request merged**